src/python/aqueduct_executor/start-function-executor.sh

```diff
@@ -11,13 +11,9 @@ if [ $EXIT_CODE != "0" ]; then exit $(($EXIT_CODE)); fi
 if test -f "$FUNCTION_EXTRACT_PATH/op/requirements.txt"
 then
       python3 -m pip freeze >> "$FUNCTION_EXTRACT_PATH/op/local_deps.txt"
-      python3 -m aqueduct_executor.operators.function_executor.prune_requirements --local_path="$FUNCTION_EXTRACT_PATH/op/local_deps.txt" --requirements_path="$FUNCTION_EXTRACT_PATH/op/requirements.txt" --missing_path="$FUNCTION_EXTRACT_PATH/op/missing.txt"
+      python3 -m aqueduct_executor.operators.function_executor.install_requirements --local_path="$FUNCTION_EXTRACT_PATH/op/local_deps.txt" --requirements_path="$FUNCTION_EXTRACT_PATH/op/requirements.txt" --missing_path="$FUNCTION_EXTRACT_PATH/op/missing.txt" --spec "$JOB_SPEC"
       EXIT_CODE=$?
       if [ $EXIT_CODE != "0" ]; then exit $(($EXIT_CODE)); fi
-      if test -f "$FUNCTION_EXTRACT_PATH/op/missing.txt"
```

Comment:
@cw75 this is where we check if the file exists, I don't think you have the equivalent check after your change